### DISCUSSION TITLE
Frontend Issue regarding to open the dashboard page

### DISCUSSION THREAD

**Discussion by mmeraj1010** [upyog/UPYOG#729] [General]
Issue:- when we open the FSM dashboard page then nothing happens Page redirects to the home page.
Nothing comes in the console and network tab.

Approaches:- 
We have tried the following things to solve the issue -
1:- As suggested by us we need to build the mono-ui for the dashboard so we have also built the same but still we can't able to open the dashboard.
2:- Then we also changed the Permission of the roles regarding the FSM dashboard and after this, we weren't able to solve the same.
we are using the following user role to the user:-

ACCESSCONTROL_ROLES_ROLES_EMPLOYEE
ACCESSCONTROL_ROLES_ROLES_FSM_ADMIN
State Administrator
ACCESSCONTROL_ROLES_ROLES_FSM_DASHBOARD_VIEWER

3:- Also we have tried to check the issue on the local we are using the same environment that we have used on the website.
I have logged in with the same account on the local which we are using on the website but in the local I dashboard option not coming in the menu so I am not able to debug the issue on the local.

Kindly help us to debug the same on the local so that we can resolve this issue.

**Comment by support-upyog** [comment]
> Issue:- when we open the FSM dashboard page then nothing happens Page redirects to the home page. Nothing comes in the console and network tab.
> 
> Approaches:- We have tried the following things to solve the issue - 1:- As suggested by us we need to build the mono-ui for the dashboard so we have also built the same but still we can't able to open the dashboard. 2:- Then we also changed the Permission of the roles regarding the FSM dashboard and after this, we weren't able to solve the same. we are using the following user role to the user:-
> 
> ACCESSCONTROL_ROLES_ROLES_EMPLOYEE ACCESSCONTROL_ROLES_ROLES_FSM_ADMIN State Administrator ACCESSCONTROL_ROLES_ROLES_FSM_DASHBOARD_VIEWER
> 
> 3:- Also we have tried to check the issue on the local we are using the same environment that we have used on the website. I have logged in with the same account on the local which we are using on the website but in the local I dashboard option not coming in the menu so I am not able to debug the issue on the local.
> 
> Kindly help us to debug the same on the local so that we can resolve this issue.

Hi

Please check below action was there in mdms,

https://github.com/upyog/upyog-mdms-data/blob/a39bbfe1aa0277a1d58ec8ae95160e0f8d9c9bad/data/pg/ACCESSCONTROL-ACTIONS-TEST/actions-test.json#L10423-L10437
